src/modules/fsresizer/ResizeFSJob.cpp

```diff
@@ -176,20 +176,20 @@ ResizeFSJob::exec()
     if ( !m.first || !m.second )
     {
         return Calamares::JobResult::error(
-            tr( "Resize Failed" ),
+            tr( "Resize Failed", "@error" ),
```

Comment:
Is capitalisation needed here? Does it look like a title?

Reply:
Via a couple of steps this kind of strings in `JobResult::error()` ends up in a pop-up dialog describing the error. The window title is "Error", there's a "Setup Failed" message at the top of the popup, and then the message is shown as *informative text* in the popup. Here, that message is "Resize Failed". So .. no, it's not a title, it is plain text and should be capitalized as normal text and end with a full stop.

Reply:
Fixed.